src/debug/validatePreDebug.ts

```diff
@@ -120,7 +155,10 @@ async function validateAzureWebJobsStorage(context: IActionContext, projectLangu
                 return new ParsedFunctionJson(await fse.readJSON(functionJsonPath));
             }));
 
-            if (functions.some(f => !f.isHttpTrigger)) {
+            // NOTE: Currently, Python V2+ requires storage to be configured, even for HTTP triggers.
+            const isPythonV2Plus = projectLanguage === 'Python' && projectLanguageModel && projectLanguageModel > 1;
```

Comment:
Can we reuse the `isPythonV2Plus` utility?

Reply:
Consolidated.